CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 -   Added a way to run CI on unmerged PRs
```

Comment:
Things to test manually:
- Sign in 
- Sign out
- Querying /api/auth/random -> 404
- Session refreshing
- Enable email verification in required mode in the app, and then sign up, do not verify email, make an API call that uses withSession function, and it should return 403 with json response body indicating the the `st-ev` claim has expired
- SSR session verification using getSSRSession
- Test withSession, but the core is down -> this should go to the next js api error handler.
- In an API that uses withSession, we want to update the session access token payload, and then we should see that in the response, we have the new access token along with the Cache-control header which is in the `addCookies` function.
- Do manual testing in astro and remix and sveltekit app as well that use these functions.

Reply:
Added all the tests and shared a video of manual tests.